Helper/Config.php

```diff
@@ -166,7 +158,7 @@ public function getEnvironmentVariable(string $envVarName): ?string
      */
     public function getAccessToken(): ?string
     {
-        return $this->encryptor->decrypt($this->scopeConfig->getValue('ns8/protect/token'));
+        return $this->encryptor->decrypt($this->scopeConfig->getValue(self::ACCESS_TOKEN_CONFIG_KEY));
```

Comment:
@cwalv Did we want it to throw somewhere if it fails to decrypt the access token?

Reply:
I think it'd be nice to figure out some way to surface that error rather than just failing silently, but I don't want to do it with this PR (I'm not sure if just throwing is really what we want to do, because if we don't catch the error further up the stack it would probably result in the order page blowing up like https://app.clubhouse.io/ns8/story/23597/magento-integration-breaks-order-view-page)